tests/e2e-tests/specs/front-end/front-end-single-product.test.js

```diff
@@ -19,7 +14,7 @@ let variablePostIdValue;
 
 describe( 'Single Product Page', () => {
 	beforeAll( async () => {
-		await activatePlugin( 'woocommerce' );
+		await StoreOwnerFlow.login();
```

Comment:
I like the change from activatePlugin to StoreOwnerFlow, makes more conceptual sense to me.

Reply:
👍